CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 Change Log
 ==========
+Version 2.2.1 *(10 April 2024)*
+-------------------------------------------
+**What's new**
+
+* **[Android Platform]**
```

Comment:
@Anush-Shand Add a note for example, update to clevertap-react-native v 2.2.1, v2.2.0 android code produces a crash.